src/LibraryManager/Providers/jsDelivr/JsDelivrCatalog.cs

```diff
@@ -155,31 +156,33 @@ private void GetFiles(JObject fileObject, List<string> result)
 
         public async Task<CompletionSet> GetLibraryCompletionSetAsync(string libraryNameStart, int caretPosition)
         {
-            CompletionSet completionSet = new CompletionSet
+            libraryNameStart = libraryNameStart == null ? string.Empty : libraryNameStart;
```

Comment:
`libraryNameStart = libraryNameStart ?? string.Empty;` is more concise.

Reply:
Although, it could also be avoided altogether with 

```
var completionSet = new CompletionSet { ... };
if (string.IsNullOrEmpty(libraryNameStart)
{
    // no point in doing the rest of the work, we know it's going to be an empty completion set anyway
    return completionSet; 
}
```